src/listen/websocket.rs

```diff
@@ -737,6 +759,10 @@ impl<'a> WebsocketHandle {
         // eprintln!("<handle> receiving response: {resp:?}");
         resp
     }
+
+    pub fn request_id(&self) -> Uuid {
```

Comment:
Add docs to this function, and please mention that the request ID is needed for support on issues that come up with a particular request.

Reply:
fe7e71d10f9d